src/UI/EnchantingWindow.cpp

```diff
@@ -30,7 +30,7 @@ void cEnchantingWindow::SetProperty(short a_Property, short a_Value, cPlayer & a
 {
 	if ((a_Property < 0) || (static_cast<size_t>(a_Property) >= ARRAYCOUNT(m_PropertyValue)))
 	{
-		ASSERT(!"a_Property is invalid");
+		LOGD("a_Property is invalid");
```

Comment:
Is this a debugging leftover?

Reply:
It might be by whoever left those broken ASSERTs there, I can remove them? or I can make them into proper asserts

Reply:
The ASSERT looks okay, why were they broken?

Reply:
ah you're right, I think I'll change them to `ASSERT(bool)` instead if `if (!bool) { assert(!message) }` though

Reply:
But then you lose the message in the alert if you just pass in a bool.

Reply:
alrighty I'll just change it back